### PR TITLE
added ComputerPage

### PR DESCRIPTION
**Pull request by avvaland26**
hi

### CODE REVIEW

**Review by avvaland26 — COMMENTED** [review]
please c

**Comment by avvaland26** [comment]
k

**Review by mital20 — APPROVED** [review]

**Review by avvaland26 — COMMENTED** [review]
hello y not

**Review by mital20 — COMMENTED** [review]
ok its done

**Review by avvaland26 — COMMENTED** [review]
#

**Review by avvaland26 — COMMENTED** [review]
- [ ] 

- [ ]

File: src/ComputerPage.java [hunk 1]
```diff
@@ -0,0 +1,2 @@
+public class ComputerPage {
```
Comment (avvaland26) [inline src/ComputerPage.java:1]: looks ok

**Review by avvaland26 — COMMENTED** [review]
approve #1

**Pull request closed without merging**